src/vic.h

```diff
@@ -20,25 +20,25 @@
 
 /* vic-serial.c */
 extern char vic_buffer[VIC_BUFFER_SIZE + 1];
-extern int vic_buffer_len;
+extern uint8_t vic_buffer_len;
```

Comment:
@vargac this effectively means that the buffer length can be at most 255 characters (+1 for `'\0`). I am good with it for the time being -- it is just something we should keep an eye on in the future.